src/dbnode/storage/limits/query_limits_test.go

```diff
@@ -314,6 +314,8 @@ func verifyMetrics(t *testing.T,
 ) {
 	snapshot := scope.Snapshot()
 
+	fmt.Println(snapshot.Gauges())
```

Comment:
Remove?

Reply:
Ah yeah, need to fix up a few of these tests